src/style.css

```diff
@@ -1 +1,155 @@
-/* add styles here */
+ @font-face {
+  font-family: Avenir-Book;
+  src: url('./fonts/Avenir-Book.ttf') format('truetype');
+  font-weight: normal;
+  font-style: normal;
+}
+
+@font-face {
+  font-family: Avenir-Heavy;
+  src: url('./fonts/Avenir-Heavy.ttf') format('truetype');
+  font-weight: normal;
+  font-style: normal;
+}
+
+* {
+  margin: 0;
+  padding: 0;
+}
+
+ .page {
+font-family: Avenir-Book, sans-serif;
+font-weight: 300;
+margin: 0;
+padding: 0;
+}
+
+.page_body {
+  margin: 0;
+}
+
+.search-bar{
+  width: 900px;
+  display: flex;
+  flex-direction: column;
+  gap: 20px;
+  align-items: center;
+  font-size: 16px;
+  line-height: 22px;
+  margin-top: 20px;
+}
+
+.search-bar__field{
+  position: relative;
+  display: flex;
+  align-items: center;
+  background-color: #fff;
+  border: 1px solid #E1E7ED;
+  border-radius: 4px;
+  box-shadow: 0 1px 8px 0 #3D4E611A;
+
+
+}
+
+.search-bar__field input {
+  width: 100%;
+  padding-left: 40px;
+  border: none;
+  outline: none;
+  background: transparent;
+  font-family: Avenir-Book, sans-serif;
+  font-size: 16px;
+
+}
+
+.search-bar__field input::placeholder{
+  font-family: Avenir,sans-serif;
```

Comment:
The `font-family` specified here is 'Avenir', which is different from the 'Avenir-Book' and 'Avenir-Heavy' defined in the `@font-face` rules. Ensure consistency in font-family naming to avoid confusion.